arch/arm/dts/imx8mm-evk-u-boot.dtsi

```diff
@@ -140,6 +140,10 @@
 	assigned-clock-parents = <&clk IMX8MM_SYS_PLL1_400M>;
 };
 
+&wdog1 {
```

Comment:
Just curious, why do we need SPL_WDT enabled? (so we need this change here because of this)

I've checked final u-boot-fio-config with both this PR and https://github.com/foundriesio/meta-lmp/pull/363 applied,
and WDT is enabled for SPL, however it's not for U-Boot proper:
```
CONFIG_SPL_WATCHDOG_SUPPORT=y
CONFIG_HW_WATCHDOG=y
CONFIG_IMX_WATCHDOG=y
CONFIG_SPL_WDT=y
# CONFIG_WDT is not set
```

I think we can just enable both (SPL_WDT, WDT) so they are consistent

Reply:
or switch both to HW_WATCHDOG/SPL_HW_WATCHDOG instead